content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,329 @@
+- Configure which aggregation you want to send to Datadog with the `histogram_aggregates` parameter in your [`datadog.yaml` configuration file][1]. By default, only `max`, `median`, `avg`, and `count` aggregations are sent out to Datadog. `sum` and `min` are also available.
 
-**Note**: A `RATE` is different from the `COUNT` metric submission type, which represents the number of events in the flush interval.
+- Configure which percentile aggregation you want to send to Datadog with the `histogram_percentiles` parameter in your [`datadog.yaml` configuration file][2]. By default, only the `95percentile` is sent out to Datadog.
 
-For instance, say that the `number.of.requests.rate` metric is reported every 30 seconds to Datadog with the `RATE` type in `server:web_1`.
 
-Each value/data point represents the "rate" of requests. The metric then reports the following values:
+[1]: https://github.com/DataDog/datadog-agent/blob/04d8ae9dd4bc6c7a64a8777e8a38127455ae3886/pkg/config/config_template.yaml#L106-L114
+[2]: https://github.com/DataDog/datadog-agent/blob/04d8ae9dd4bc6c7a64a8777e8a38127455ae3886/pkg/config/config_template.yaml#L116-L121
+{{% /tab %}}
+{{% tab "DISTRIBUTION" %}}
 
-* `0.33` for the first 30 seconds
-* `0.66` for the second interval of 30 seconds
-* `null` for the last interval of 30 seconds
-Then this pattern of `0.33`, `0.66`, `0`, repeats. **Note**: for a `RATE` metric, when a `0` value is submitted, `null` is stored within Datadog.
+The DISTRIBUTION metric submission type represents the global statistical distribution of a set of values calculated across your entire distributed infrastructure in one time interval. A DISTRIBUTION can be used to instrument logical objects, like services, independently from the underlying hosts.
 
-Since the `RATE` is the normalized per-second variation of the number of requests. When graphed, this `RATE` metric looks like the following:
+Unlike the HISTOGRAM metric type, which aggregates on the Agent during a given time interval, a DISTRIBUTION metric sends all the raw data during a time interval to Datadog, and aggregations occur server-side. Because the underlying data structure represents raw, unaggregated data, distributions provide two major features:
 
-{{< img src="developers/metrics/types/rate_metric.png" alt="Rate Metric" >}}
+- Calculation of percentile aggregations
+- Customization of tagging
 
-Discover how to submit rate metrics:
+If you send `X` values for a DISTRIBUTION metric `<METRIC_NAME>` in a given time interval, the following aggregations are available for query by default:
 
-* [With a custom Agent Check][1]
-* [With the Datadog API][2]
+| Aggregation           | Description                                                                                                                                          | Datadog In-app Type |
```

Comment:
| Aggregation           | Description                                                                                                                                          | Datadog In-App Type |